applications/nublado/values-ccin2p3.yaml

```diff
@@ -64,11 +69,22 @@ controller:
         - containerPath: "/sps"
           volumeName: "sps"
 jupyterhub:
+  prePuller:
+    continuous:
+      enabled: true
```

Comment:
This won't do anything of use so far as I can tell. I think it will try to pre-pull a stock JupyterLab image, which Nublado doesn't provide a mechanism to launch, so I think it will just waste disk space without accomplishing anything.